Update garbage-collection.md (#47901)
kubernetes/website: content/en/docs/concepts/architecture/garbage-collection.md

As a beta feature, you can specify the maximum time a local image can be unused for,
regardless of disk usage. This is a kubelet setting that you configure for each node.

> To configure the setting, we just need to set a value for the `imageMaximumGCAge` field in the kubelet configuration file.

Reviewer: ```suggestion
To configure the setting, you need to set a value for the `imageMaximumGCAge` field
in the kubelet configuration file.
```
